Blog post re: names and structure (#1502)
codatio/codat-docs: blog/250204-new-product-docs.md

---
title: "Introducing Codat's solution updates"
date: "2025-02-04"
tags: ["Product", "Update"]
authors: pzaichkina
---
import Products from "@components/Products";

Codat's solutions and documentation got a refresh - smarter navigation, clearer structure, and with a focus on what matters most. 

<!--truncate-->

## What's new?

> As we focus on supporting you in building deeper connections with your business customers, we evolved our solutions and reorganized our documentation so you can find the support you need, fast.

Reviewer: ```suggestion
As we focus on supporting you in building deeper connections with your business customers, we have evolved our solutions and reorganized our documentation so that you can find the support you need, fast.

```

> Each of Codat's solutions is tailored to a specific use case we're experts in, so our homepage and navigation structure put these flagship use cases front and center. 

Reviewer: ```suggestion
Each of Codat's expert solutions is tailored to a specific use case, so our homepage and navigation structure put these flagship use cases front and center. 

```

> Our range of solutions also boasts updated, clearer names while delivering the same reliable insights and best-in-class experience you are used to. 

Reviewer: I'd just say we've also renamed the solutions for clarity or something

---
title: "Updates to Codat's solution range"
date: "2025-02-04"
tags: ["Product", "Update"]
authors: pzaichkina
---
import Products from "@components/Products";

Codat's solutions and documentation got a refresh - clearer naming, smarter navigation, now with a focus on what matters most. 

<!--truncate-->

Codat exists to transform how financial institutions help businesses grow and thrive. We are proud of the products we have built for software developers that raise the bar in financial services for small and medium-sized businesses.

As we have continued on this journey, we have built solutions for client-facing teams, simplifying their processes and delivering deeper and more actionable insights into business data. 

The changes to our solution offering and documentation reflect this evolution.

## What's new?

As we focus on supporting you in building deeper connections with your business customers, we evolved our solutions and reorganized our documentation so you can find the support you need, fast.

Each of Codat's solutions is tailored to a specific use case we're experts in, so our homepage and navigation structure put these flagship use cases front and center. 

> Our range of solutions also boasts updated, clearer names while delivering the same reliable insights and best-in-class experience you are used to. 

Author: ```suggestion
We have also renamed our range of solutions to reflect their focus clearly while delivering the same best-in-class experience you are used to. 
```
